py/torch_tensorrt/dynamo/conversion/impl/slice/ops.py

```diff
@@ -262,6 +262,75 @@ def flip(
     return layer.get_output(0)
 
 
+def calculate_strides(shape: Sequence[int]) -> Sequence[int]:
+    strides = [1] * len(shape)
+    for i in range(len(shape) - 2, -1, -1):
+        strides[i] = strides[i + 1] * shape[i + 1]
+    return strides
```

Comment:
Is this function a general util or just specific for this case? If general, I think it would be perfect to move it into the `converter_utils.py` with detailed explanations on limitations and how to use it. Otherwise, it could be moved into the function of `diagonal`

Reply:
Yes, this function can be a general utility function. Therefore, I have moved the `calculate_strides` function to `converter_utils.py`. Thank you for the useful comment.